### ISSUE TITLE
Can't find documentation on Microsoft.VisualStudio.ReleaseManagement api?

### ISSUE THREAD

**Issue by msyrak** [dotnet/dotnet-api-docs#9348]
I'm trying to find the documentation for Microsoft.VisualStudio.ReleaseManagement,WebAPI but it is really hard. Can you help me?

**Comment by krwq** [comment]
Will https://learn.microsoft.com/en-us/azure/devops/integrate/concepts/dotnet-client-libraries?view=azure-devops help? I'd generally recommend to search for specific REST APIs

**Comment by msyrak** [comment]
Hi Krzys 
Yes, that link was the one that deceived me every time, because I thought, "There it is!". But then all i get was a package install guide when clicking the link.
Later i found the REST API documentation, and that helped me out, so your recommendation is spot on.
It would, however, be nice with a link to the respective REST api's documentation on https://learn.microsoft.com/en-us/azure/devops/integrate/concepts/dotnet-client-libraries?view=azure-devops.

**Comment by krwq** [comment]
I agree it would be nice to add this - I'll leave this issue open for now because I'm not quite sure where to redirect this issue. @mairaw any clues?
